package.json

```diff
@@ -41,6 +41,7 @@
   "dependencies": {
     "chalk": "^2.4.2",
     "config-shield": "^0.2.0",
+    "extend": "^3.0.2",
```

Comment:
I am not sure why, but without explicitly adding the dependency here, I got error on the following lines in code. 
`const extend = require('extend');`